client/components/deposits-overview/index.tsx

```diff
@@ -23,14 +23,11 @@ const DepositsOverview = (): JSX.Element => {
 	const overview = currencies[ 0 ]; // TODO: To handle multiple currencies we'll need to fetch the currently selected currency.
 
 	return (
-		<Card>
+		<Card className="wcpay-deposits-overview">
 			<CardHeader>{ strings.heading }</CardHeader>
 			<NextDepositDetails isLoading={ isLoading } overview={ overview } />
 
 			<p>Deposits History Section Goes here</p>
-
-			<p>Deposits Card Footer/Action Goes here</p>
-
```

Comment:
I removed this while here, we have the footer implemented now! `<DepositsOverviewFooter />`